scripts/datacontainer/run.sh

```diff
@@ -0,0 +1,18 @@
+
+#!/bin/bash
+
+timestamp=`date +%Y/%m/%d-%H:%M:%S`
+echo "[$timestamp] Copying files to /usr/data/openapispecs"
+mkdir -p /usr/data/openapispecs/.git
```

Comment:
No need to mkdir first. just cp -R /usr/data/specrepo/.git /usr/data/openapispecs/